src/components/gallery/gallery-item.tsx

```diff
@@ -1,11 +1,10 @@
 type GalleryItemProps = {
   src: string;
-  alt: string;
 }
-function GalleryItem ({src, alt}: GalleryItemProps) : JSX.Element {
+function GalleryItem ({src}: GalleryItemProps) : JSX.Element {
```

Comment:
здесь по сути можно передать стандартные пропсы для image

```suggestion
function GalleryItem ({src}: HTMLImageElement) : JSX.Element {
```

Таким образом мы получим доступ ко всем пропсам img для компонента